packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import OrgNav from './org-nav/index';
+import ProjectNav from './project-nav/index';
+import {
+  DataInterface,
+  Product,
+  URLSInterface,
+  HostsInterface,
+  NavItem,
+} from './types';
+
+interface MongoNavInterface {
+  activeProduct: Product;
+  activeNav?: NavItem;
+  admin?: boolean;
+  data: DataInterface;
+  onOrganizationChange: React.ChangeEventHandler;
+  onProjectChange: React.ChangeEventHandler;
+  constructOrganizationURL?: (orgID: string) => string;
+  constructProjectURL?: (orgID: string, projID: string) => string;
+  showProjNav?: boolean;
+  hosts?: HostsInterface;
+  urls?: URLSInterface;
+}
+
+export default function MongoNav({
+  activeProduct,
+  activeNav,
+  data,
+  hosts,
+  onOrganizationChange,
+  onProjectChange,
+  urls,
+  showProjNav = true,
+  admin = false,
+  constructOrganizationURL = orgId =>
+    `https://cloud.mongodb.com/v2#/org/${orgId}/projects`,
+  constructProjectURL = (orgId, projectId) =>
+    `https://cloud.mongodb.com/v2#/${projectId}`,
+}: MongoNavInterface) {
+  const accountHost = hosts?.account ?? `https://account.mongodb.com`;
```

Comment:
Is there an alternative you'd propose? Looking at the built `index.web.js` file, it looks less egregious to me:

```js
const accountHost = hosts?.account ?? `https://account.mongodb.com`;
// to
Ee = null !== (t = null == fe ? void 0 : fe.account) && void 0 !== t ? t : "https://account.mongodb.com",
```

Ultimately whether or not we're okay with this is based on what would be shipped to the user. I'm not convinced that the bundle would be smaller if we wrote this differently.

That said, the `index.web.js` file altogether looks like it's about 60KB (20K gzipped), which seems a tad large. I do think it's partially because of the SVG paths that are being bundled (which should decrease in this package once the final versions are included in the Icon component).

I wonder if there are any quick wins in terms of bundle size we could make here...

Reply:
I suppose the fixture data we're using won't be in v1 either.